charts/enterprise/authelia/questions.yaml

```diff
@@ -433,34 +451,34 @@ questions:
                       description: "The groups filter used in search queries to find the groups of the user."
                       schema:
                         type: string
-                        default: ""
+                        default: "(member={dn})"
                         required: true
                     - variable: group_name_attribute
                       label: "Group name Attribute"
                       description: "The attribute holding the name of the group"
                       schema:
                         type: string
-                        default: ""
+                        default: "cn"
                         required: true
                     - variable: mail_attribute
                       label: "Mail Attribute"
                       description: "The attribute holding the primary mail address of the user"
                       schema:
                         type: string
-                        default: ""
+                        default: "mail"
                         required: true
                     - variable: display_name_attribute
                       label: "Display Name Attribute"
                       description: "he attribute holding the display name of the user. This will be used to greet an authenticated user."
                       schema:
                         type: string
-                        default: ""
+                        default: "displayName"
                     - variable: user
                       label: "Admin User"
                       description: "The username of the admin user used to connect to LDAP."
                       schema:
                         type: string
-                        default: "CN=Authelia,DC=example,DC=com"
+                        default: "uid=admin,ou=people,DC=example,DC=com"
```

Comment:
I'm not an LDAP guy, but above we have OU in caps, but here not. Does it matter?

Reply:
doesn't really mater afaik.